README.md

```diff
@@ -26,6 +26,18 @@ environments, develop in SNAFU and then write ripsaw benchmark to integrate with
 | Elasticsearch  | Working  |
 | Prom           | Planned  |
 
+
+It is suggested to use a venv to install and run snafu.
+
+```
+python3 -m venv /path/to/new/virtual/environment
```

Comment:
Can you please also add documentation to show snafu can be imported as a python library and run a benchmark wrapper?

Reply:
this pr enables to use it as a binary. Will create a new pr to support using as a library too.